site/public/css/rainbow-customization.css

```diff
@@ -212,3 +212,13 @@ hr {
 .override + div.gradeable-li-right-of-input i.fa-exclamation-triangle {
     display: inline-block;
 }
+
+.input-field {
+    display: flex;
+    align-items: center;
+}
+
+/* stylelint-disable-next-line selector-class-pattern */
+div#gradeables.customization_item div input[type="text"] {
+    width: 50px;
+}
```

Comment:
It would be better to use a class to identify these input fields instead of this hacky selector.